academic-services/knowledge-exploration-service/reference-post-interpret.md

```diff
@@ -0,0 +1,473 @@
+---
+title: POST Interpret
+description: Generates semantic interpretations of a natural language query
+ms.topic: reference
+ms.date: 2020-02-07
+---
+
+# Interpret REST API
+
+The **Interpret** method generates semantic interpretations of a natural language query based on the combination of a language grammar and indexed attributes.
+
+It can be used as either a way of interpreting a specific natural language query, or to generate natural language query suggestions using the original query as a stem.
+
+For full details on the language grammar see the [natural language queries](concepts-queries.md) page.
+
+``` HTTP
+POST http://{serviceName}.{serviceRegion}.cloudapp.azure.com/interpret
+```  
+
+## Request Body
+
+Name | Required | Type | Description
+--- | --- | --- | ---
+`query` | Required | string | The natural language query to generate interpretations for. See the [natural language queries](concepts-queries.md) page for more details.
+`count` | Optional | integer | The number of interpretations to generate in the result set. <br/><br/>Defaults to 5.
```

Comment:
>The number of interpretations to generate in the result set. <br/><br/>Defaults to 5. [](start = 31, length = 85)

Per Charles' other comment:
"The maximum number of interpretations to generate"